python/docs/_extensions/machow/interlinks/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.html
```

Comment:
What's the story on the _extensions? Is it basically vendoring of the doc dependencies? It may be helpful to drop a `python/docs/_extensions/README.md` to explain the directory / rationale for what's in here and how/when to update?